### DISCUSSION TITLE
Baseprint JATS

### DISCUSSION THREAD

**Discussion by castedo** [singlesourcepub/community#53] [Ideas]
Here is a proposal for a new set of documents somewhere under https://singlesource.pub.

@tarleb and @xldrkp, what do you think of there being a new github.com/singlesourcepub/baseprints repository for this documentation project?
singlesourcepub seems like a great online forum for this activity.

The purpose of these documents is to facilitate the establishment of a new de facto standard supported by FOSS (Free Open-Source Software).  I propose that these documents are more descriptions of real FOSS functionality, and less prescriptive specifications. More like caniuse.com and less like a W3C or NISO specification. Functionality that is not yet implemented in FOSS goes into documents called proposals (like Python Enhancement Proposals). I have some additional related plans which I can describe in the new repository.

The focus of this documentation is on specific usage scenarios. "Baseprint" is a convenient name for the critical open data format in theses usage scenarios.  Thanks to those who voted and gave feedback on the name. The closest existing thing to a baseprint is an open access PubMed Central JATS package. There are millions of these currently archived and available. In large part for this reason, JATS is the basis for the new open data format for baseprints (pun intended).

"Baseprint JATS" is the name I'll start using until a better clever acronym is found. The chair for JATS4R @Melissa37 convinced me that my first acronym idea, JATS4B, will cause undesirable confusion. I think she's right.

Two important features of a baseprint are that it:

A) can be converted automatically without human intervention into a PDF and HTML content using pure FOSS, and
B) is archived for long-term preservation and potential use in the distant future.

It is a non-goal of Baseprint JATS to be adopted by processing pipelines requiring proprietary software and non-automated human intervention to produce PDF files and HTML content. If a digital object can only be converted to PDF or HTML by a proprietary pipeline then it is by definition not a baseprint and not in Baseprint JATS format.

**Comment by xldrkp** [comment]
Hi @castedo, this sounds like a great idea to me. What do you need for setting this up here?
